Formula/e/erlang@25.rb

```diff
@@ -23,6 +23,9 @@ class ErlangAT25 < Formula
 
   keg_only :versioned_formula
 
+  # EOL with OTP-28 release (2025-05-17)
+  disable! date: "2025-05-25", because: :unsupported
```

Comment:
Let's do it like this to give people time to do something about it.
```suggestion
  # EOL with OTP-28 release (2025-05-17)
  deprecate! date: "2025-05-17", because: :unsupported
  disable! date: "2025-08-17", because: :unsupported
```

Reply:
Because of there is a dependent but it will be disabled before erlang@24 and that is fine to go.